package.json

```diff
@@ -1,33 +1,29 @@
 {
   "name": "api-schema-builder",
-  "version": "1.0.1",
+  "version": "1.0.3",
   "description": "build schema with validators for each endpoint",
   "main": "src/index.js",
+  "files": [
+    "src"
+  ],
   "nyc": {
     "report-dir": "./coverage",
     "lines": 95,
     "statements": 95,
     "functions": 100,
     "branches": 90,
-    "include": [
-      "src/*.js",
-      "src/*/*.js"
-    ],
     "reporter": [
       "lcov",
       "text"
     ],
     "cache": true,
     "all": true
   },
-  "files": [
-    "src"
-  ],
   "license": "Apache-2.0",
   "scripts": {
-    "test": "node_modules/mocha/bin/_mocha ./test/*/*-test.js --recursive",
-    "test:coverage": "nyc node_modules/mocha/bin/_mocha --recursive ./test/*/*-test.js ",
-    "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls",
+    "test": "node_modules/mocha/bin/_mocha ./test/*/*-test.js ./test/*/*/*-test.js --recursive",
```

Comment:
You may start with `mocha` no need for its path.
For example:
`    "test": "mocha ./tests/unit-tests --recursive",
    "unit-tests": "nyc --produce-source-map npm test",`

Reply:
I will check that num of tests are equal

Reply:
there are the same amount of tests